pkgs/by-name/si/sink-rotate/package.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pipewire
+, wireplumber
+}:
+let
+  version = "1.0.1";
+in
+rustPlatform.buildRustPackage {
+  pname = "sink-rotate";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "mightyiam";
+    repo = "sink-rotate";
+    rev = "v${version}";
+    hash = "sha256-WwHrmCoFNndM8AZUFdhR6qVgzmkXtdJPB6L/0cjtLi0=";
+  };
+
+  cargoHash = "sha256-anMFInvSm3LZT8dSek6o324Wy+dAAH1q2shIs7Kak1o=";
+
+  buildInputs = [ pipewire wireplumber ];
+
+  meta = with lib; {
+    description = ''Command that sets "next" PipeWire audio sink as default'';
+    homepage = "https://github.com/mightyiam/sink-rotate";
+    license = licenses.mit;
+    maintainers = with maintainers; [ mightyiam ];
```

Comment:
Could you add both the [mainProgram](https://nixos.org/manual/nixpkgs/stable/#var-meta-mainProgram) and [platforms](https://nixos.org/manual/nixpkgs/stable/#var-meta-platforms) meta attributes?

Reply:
Done. Regarding platforms, I didn't get deep into it. I copied the value from the pipewire package.

Reply:
I've found out that `buildRustPackage` [already sets it up](https://github.com/NixOS/nixpkgs/blob/754ada3a56a6df53a1c7a2941dbba0bc63df040f/pkgs/build-support/rust/build-rust-package/default.nix#L155), so, usually, there's no need to add the meta attribute `platforms` when using `buildRustPackage`.

Reply:
since this depends on pipewire it seems we should restrict it to linux only